### ISSUE TITLE
bit artifacts command 

### ISSUE THREAD

**Issue by GiladShoham** [teambit/bit#4541]
This new command will give you a way to easily list/download artifacts from components

## List artifacts
`bit artifacts <comp-id>` will show all available artifacts (relative paths of them) for the component.
It will be grouped by the aspect then the task that created the artifacts

### example:
```
teambit.compilation/babel (title)
BabelCompiler (teambit.compilation/compiler) (subtitle)

dist/index.js
dist/index.d.ts
dist/button.js

teambit.preview/preview
GeneratePreview (teambit.preview/preview)

public/index.html
public/static/js/main.993f88ba.js
```

## Download artifacts
`bit artifacts <comp-id-with-star-support> --aspect <aspect-id> --task <task-id> --files <glob> --out-dir <dir path>`
dir path will support special template like {comp-id}

## Visual example
It should be something similar to this, but from terminal
![image](https://user-images.githubusercontent.com/3840769/124904213-73531500-dfed-11eb-815f-2ba31e46afe8.png)

**Comment by itaymendel** [comment]
* should support fetching from many components as well (several params or glob-pattern)

**Comment by GiladShoham** [comment]
> * should support fetching from many components as well (several params or glob-pattern)

Agree, that's why the `dir path will support special template like {comp-id}` comes into place.
(editing the original description to specify that comp-id is optional)

**Comment by davidfirst** [comment]
Implemented by the above PR.
I support both: entering multiple components and components with patterns.
One thing was not implemented is this:

> dir path will support special template like {comp-id}

See the screenshot I uploaded in the PR above. I think the directory structure there is good for 95% of the cases. It's very convenient as it groups your scopes/namespaces and then it saves according to the original path. We can improve it later.
